config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -577,6 +577,42 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-akeyless
       description: "Run e2e test with akeyless provider for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-conjur
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    always_run: false
+    optional: true
+    path_alias: sigs.k8s.io/secrets-store-csi-driver
+    branches:
+    - ^main$
+    - ^release-*
+    labels:
+      # this is required because we want to run kind in docker
+      preset-dind-enabled: "true"
+      # this is required to make CNI installation to succeed for kind
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240409-13cd3acf7e-master
+        command:
+          - runner.sh
+        args:
+          - bash
+          - -c
+          - >-
+            make e2e-bootstrap e2e-helm-deploy e2e-conjur
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: "4"
+            memory: "4Gi"
```

Comment:
Use the same memory limits as other provider e2e jobs

https://github.com/kubernetes/test-infra/blob/dcb0d8161491edec282e502ed2160c3d93567692/config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml#L167-L173